lib/client/backend_client.dart

```diff
@@ -23,8 +22,25 @@ class BackendClient {
     }
   }
 
-  void postData(String data) {
-    //add task to API @PostRequest
+  Future<dynamic> postData({required String uri, dynamic body}) async {
+    try {
+      var url = Uri.http(localhost, uri);
+      final response = await http.post(
+        url,
+        headers: headers,
+        body: jsonEncode(body),
+      );
+      if (response.statusCode == 201) {
```

Comment:
I would recommend using the `enum` that Flutter provides, but if this number literal works better for your brain, "201" works as well. Feel free to resolve this comment.

Untested suggestion:
```suggestion
      if (response.statusCode == HttpStatus.created) {
```

Note: `HttpStatus` can come from `dart:html` or `dart:io`. I think `dart:io` is better for non-web projects.

Reply:
done